pom.xml

```diff
@@ -143,18 +143,19 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                      <phase>verify</phase>
-                      <goals>
-                        <goal>exec</goal>
-                      </goals>
+                       <id>create-sevntu-patch</id>
+                       <phase>verify</phase>
+                       <goals>
+                          <goal>exec</goal>
+                       </goals>
                     </execution>
                 </executions>
                 <configuration>
                     <executable>git</executable>
                     <arguments>
-                      <argument>diff</argument>
-                      <argument>HEAD~1</argument>
-                      <argument>HEAD</argument>
+                       <argument>diff</argument>
+                       <argument>HEAD~1</argument>
+                       <argument>HEAD</argument>
                     </arguments>
```

Comment:
This is now +3 over `</arguments>`, but `<arguments>` is +4 over `<configuration>`. `<executions>` is +4 over `<execution>`. I would make this all consistent.

Reply:
Not sure how that escaped my 👀 . Fixed. In fact, I now looked up the Maven coding conventions, and they want 2 spaces. I'll put automated formatted of the POM on my todo list. :)